src/apply.ts

```diff
@@ -17,12 +17,11 @@
 
 import type * as stream from 'stream';
 import * as resourceBundle from '@balena/resource-bundle';
-import * as SDK from 'balena-sdk';
+import type * as SDK from 'balena-sdk';
 
-interface ReadOptions {
-	apiUrl: string;
+interface ApplyOptions {
 	application: number;
-	authToken: string;
+	sdk: SDK.BalenaSDK;
```

Comment:
You’re changing the API so this technically is a breaking change which would be denoted by a major version bump if we were post-v1.0, but needs to be a minor since we’re pre-v1.0.

Reply:
Right.  I amended the commit to update the change type.